File: Java Programs (BlueJ)/Class 10/time.java
import java.util.*;
class time
{
    public static void main(String args[])
    {
        int h1,h2,h3,m1,m2,m3,m4,c=0;
        Scanner sc=new Scanner (System.in);
 
        System.out.println("ENTER FIRST TIME INTERVAL");
        System.out.println("ENTER HOUR");
        h1=sc.nextInt();
        System.out.println("ENTER MINUTES");
        m1=sc.nextInt();
        System.out.println("ENTER SECOND TIME INTERVAL");
        System.out.println("ENTER HOUR");
        h2=sc.nextInt();
        System.out.println("ENTER MINUTES");
        m2=sc.nextInt();
        m3=m1+m2;
        h3=h1+h2;
        if(m3>60)
        {
            
            while(m3>=60)
            {
            m3=m3-60;
            c++;
            }
            h3=h3+c;
            System.out.println("TOTAL TIME= "+h3+ " hours "+m3+" mins ");
         }
         else
            {
                System.out.println("TOTAL TIME= "+h3+ " hours "+m3+" mins ");
            }
        }
    }

    

File: MCA Sem 2/C++ codes/time.cpp
#include<iostream>
using namespace std;
class TIME{
    int hours;
    int minutes;
    int seconds;
    public:
        TIME(){
            hours = 0;
            minutes = 0;
            seconds = 0;
        }
        TIME(int h,int m,int s){
            hours = h;
            minutes = m;
            seconds = s;
        }
        void add(TIME,TIME);
        void displayTime(){
            cout<<hours<<":";
            cout<<minutes<<":";
            cout<<seconds<<endl;
        }
};
void TIME::add(TIME T1,TIME T2){
    seconds = T1.seconds + T2.seconds;
    minutes = T1.minutes + T2.minutes +seconds/60;
    hours = T1.hours + T2.hours + minutes/60;
    minutes = minutes%60;
    seconds = seconds%60;
}
int main(){
TIME T1(2,45,50);
TIME T2(1,20,30);
TIME T3;
T3.add(T1,T2);
T1.displayTime();
T2.displayTime();
T3.displayTime();
}